Formula/volta.rb

```diff
@@ -0,0 +1,27 @@
+class Volta < Formula
+  desc "Volta: The JavaScript Launcher"
+  homepage "https://volta.sh"
+  url "https://github.com/volta-cli/volta.git", :revision => "10a9bee148d2a833069d6cc27aea56af3092dc6e"
+  version "0.7.2"
+
+  depends_on "rust" => :build
+
+  conflicts_with "hadoop", :because => "both install `yarn` binaries"
+  conflicts_with "node", :because => "both install `node` binaries"
+  conflicts_with "npm", :because => "both install `npm` binaries"
+  conflicts_with "yarn", :because => "both install `yarn` binaries"
+
+  def install
+    ENV["VOLTA_HOME"] = prefix.to_s
+    system "./dev/unix/volta-install.sh", "--release", "--skip-setup"
+  end
+
+  def postinstall
+    chmod "+x", "#{opt_prefix}/bin/volta"
+  end
+
+  test do
+    system "#{bin}/volta", "install", "node@12.16.1"
+    assert_match "#{ENV["HOME"]}/.volta/tools/image/node/12.16.1/6.13.4/bin/node", shell_output("#{bin}/volta which node")
```

Comment:
`node/12.16.1/6.13.4/bin/node` seems pretty brittle.

Reply:
This is not brittle in a general sense—part of Volta's guarantee is determinism—but it *is* subject to breakage in that the layout of Volta's internal paths is private and we reserve the right to change it at any time, so I'd agree with the basic assessment here. A better test would likely be invoking effectively (my scripting fu is weak right now) `(volta which node) --version` and confirming its output is as expected.

Reply:
> `(volta which node) --version` and confirming its output is as expected.

This seems like a better idea, yeh 👍

Reply:
I changed the test to checking the version output instead of the `PATH` itself. Resolved on d142d754e